package.json

```diff
@@ -114,7 +114,7 @@
     "screwdriver-executor-queue": "^4.0.0",
     "screwdriver-executor-router": "^3.0.0",
     "screwdriver-logger": "^2.0.0",
-    "screwdriver-models": "^29.18.4",
+    "screwdriver-models": "file:../models/screwdriver-models-29.0.0.tgz",
```

Comment:
TODO: Update the version after https://github.com/screwdriver-cd/models/pull/627 is merged